devices/devs.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-loader
+ *
+ * Devices Interface
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _DEVS_H_
+#define _DEVS_H_
+
+#include "../types.h"
+#include "config.h"
```

Comment:
is `config.h` needed for this header to parse?

Reply:
`config.h` contains platform specific typedefs. In this case, the `ssize_t` is located in config header.